bin/oneoff/update_app_ids_in_enrollments.rb

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+
+puts "Updating application_id in enrollments ...\n\n"
+
+# The method set_application_id is triggered before a save
+# Save the enrollment to update set_application_id
+Pd::Enrollment.where.not(user_id: nil).each(&:save)
```

Comment:
nit: use `save!` here. `save!` will throw an error if something goes wrong whereas `save` returns a boolean, which could cause silent errors in this case

Reply:
Ah good call, thanks! Just updated it